app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -78,6 +78,7 @@ ns.SearchPage = do (ko
 
       @bindingsLoaded = ko.observable(false)
       @labs = ko.observable(false)
+      @showFocusedCollections = ko.observable(true)
```

Comment:
Does this mean we can only show or hide all layers? What defines 'focused' in this context?

Reply:
I don't know what this means, but the current Prod EDSC does have the show/hide all layers/collections on the project page. It would be nice if we kept that functionality. It would require a new button on the E2E project page though as the new one doesn't have that button (you have to do them individually).